src/legacy/core_plugins/vis_type_table/public/table_vis_fn.test.ts

```diff
@@ -24,15 +24,23 @@ import { functionWrapper } from '../../interpreter/test_helpers';
 
 jest.mock('ui/new_platform');
 
-const mockResponseHandler = jest.fn().mockReturnValue(
-  Promise.resolve({
-    tables: [{ columns: [], rows: [] }],
-  })
-);
+jest.mock('ui/vis/response_handlers/legacy', () => {
```

Comment:
The jest errors we were due to a change in way that modules were being referenced and loaded. fun fact - jest hoists `jest.mock` calls to the top. Also, @spalger is alright in my book